cmd/globals.go

```diff
@@ -123,8 +123,8 @@ var (
 	globalHTTPServerErrorCh = make(chan error)
 	globalOSSignalCh        = make(chan os.Signal, 1)
 
-	// Enable HTTP request/response headers and body logging.
-	globalHTTPTrace bool
+	// File descriptor pointing to a file where HTTP request/response headers and body are logged.
```

Comment:
you can just say "File to log HTTP request/response..." because File-descriptor will mean 0, 1, 2 etc.

Reply:
done